packages/cli/src/lib/onboarding/local.ts

```diff
@@ -49,7 +49,8 @@ export async function setupLocalStarterTemplate(
         message: 'Connect to Shopify',
         choices: [
           {
-            label: 'Use sample data from Mock.shop (no login required)',
+            label:
+              'Use sample data from mock.shop (set up shop connection later)',
```

Comment:
Since we talk about their `Shopify account` in the second option, might be helpful to use the same language here:

```suggestion
              'Use sample data from mock.shop (You can connect a Shopify account later)',
```